frontend/src/components/project/Buttons/GoBackButton.js

```diff
@@ -0,0 +1,67 @@
+import { Button, IconButton, makeStyles } from "@material-ui/core";
+import ArrowBackIcon from "@material-ui/icons/ArrowBack";
+import React from "react";
+import useElementProps from "../../hooks/useElementProps";
+import ArrowBackIosIcon from "@material-ui/icons/ArrowBackIos";
+import { useRouter } from "next/router";
+
+const useStyles = makeStyles((theme) => ({
+  root: (props) => ({
+    position: "absolute",
+    height: props.tinyScreen ? 0 : props.containerHeight,
+    top: props.tinyScreen ? 0 : props.containerTop,
```

Comment:
Why are you using position `absolute` and these hook-calculated offsets here?
It seems like this should also be possible with just using display: `flex` on the parent element on desktop - and for mobile you don't seem to need the calculation.

Reply:
Thank you for your feedback. I attempted to implement it (see recent commits). I hope the code is more suitable now. I'm still happy to take further feedback! :)